doc/sphinx/changes.rst

```diff
@@ -853,6 +853,64 @@ Changes in 8.12+beta1
   (`#9803 <https://github.com/coq/coq/pull/9803>`_,
   by Laurent Théry and Michael Soegtrop).
 
+**Reference manual**
+
+- **Changed:**
+  The reference manual has been restructured.  In the new version,
+  there are less top-level chapters, and, in the HTML format, chapters
+  are split on multiple pages.  This is still a work-in-progress and
+  further restructuring is expected in the next versions of Coq
+  (`CEP#43 <https://github.com/coq/ceps/pull/43>`_, implemented in
+  `#11601 <https://github.com/coq/coq/pull/11601>`_,
+  `#11871 <https://github.com/coq/coq/pull/11871>`_,
+  `#11914 <https://github.com/coq/coq/pull/11914>`_,
+  `#12148 <https://github.com/coq/coq/pull/12148>`_,
+  `#12172 <https://github.com/coq/coq/pull/12172>`_,
+  `#12239 <https://github.com/coq/coq/pull/12239>`_
+  and `#12330 <https://github.com/coq/coq/pull/12330>`_,
+  effort inspired by Matthieu Sozeau, led by Théo Zimmermann, with
+  help and reviews of Jim Fehrle, Clément Pit-Claudel and others).
+- **Changed:**
+  Most of the grammar is now presented using the same box mechanism as
+  was already used to present commands and tactics since Coq 8.8 and
+  which is documented in :ref:`syntax-conventions`
```

Comment:
```suggestion
  Most of the grammar is now presented using the notation mechanism that
  has been used to present commands and tactics since Coq 8.8,
  which is documented in :ref:`syntax-conventions`
```